### ISSUE TITLE
[Question] Is there a version of the management library for graph that makes use of the Microsoft Graph Api

### ISSUE THREAD

**Issue by kbulte** [Azure/azure-libraries-for-net#665]
I tried the Microsoft.Azure.Management.Graph.RBAC.Fluent library and I noticed this is still using the legacy Azure Active Directory Graph (https://graph.windows.net/). Is there already a version of the Microsoft.Azure.Management.Graph.RBAC.Fluent library that uses the Microsoft Graph api (https://graph.microsoft.com/)?

**Comment by kurtzeborn** [comment]
Thank you for opening this issue! We are routing it to the appropriate team for follow up.

**Comment by qub1n** [comment]
Yes, please replace deprecated API
https://developer.microsoft.com/en-us/office/blogs/microsoft-graph-or-azure-ad-graph/

**Comment by xccc-msft** [comment]
@kbulte No, we don't have a version using Microsoft Graph now as the service team haven't provided new swagger file for consuming Microsoft Graph APIs.

**Comment by xccc-msft** [comment]
Close this as we don't support this feature.
